extensions/amp-google-assistant-assistjs/validator-amp-google-assistant-assistjs.protoascii

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-assistant-assistjs
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-assistant-assistjs"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-assistant-assistjs>
+  html_format: AMP
+  tag_name: "AMP-ASSISTANT-ASSISTJS"
+  requires_extension: "amp-google-assistant-assistjs"
```

Comment:
This doesn't match the extension name, so probably will give you an error.

Reply:
Done